src/shared-components/EventForm/EventForm.module.css

```
@@ -37,32 +37,26 @@
-  --events-gap: 1rem;
-  gap: var(--events-gap, 1rem);
-}
-
-.checkboxdivEvents > label {
-  margin-right: 50px;
-}
 
-.checkboxdivEvents > label > input {
-  margin-left: 10px;
+  --events-gap: var(--space-5);
+  column-gap: var(--events-gap, 1rem);
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)